spec/advance_enumerable_spec.rb

```diff
@@ -0,0 +1,163 @@
+require '../advace_enumerable.rb'
+RSpec.describe Enumerable do
+  include Enumerable
+  let(:arr) { [2, 1, 7] }
+  let(:arr_nil) { [false, nil, false] }
+  let(:multiply) do
```

Comment:
You should include other types of enumerables on your tests. Like ranges, arrays of strings, a string, etc.